File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{

    int n;
    int j;
    int i;
    int k;

    do // Prompt height within range
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    for (i = 0; i < n; i++) // Loop for rows
    {
        for (k = n; k > i + 1; k--) // Loop for spaces
        {
            printf(" ");
        }
        for (j = 0; j < i + 1; j++) // Loop for hashes
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario/less/mario.py
from cs50 import get_int

while True:
    block = get_int("Height: ")
    if block > 0 and block < 9:
        break

increase = 1
decrease = block - 1

for i in range(block):
    for j in range(decrease):
        print(" ", end = "")
    for j in range(increase):
        print("#", end = "")
    increase += 1
    decrease -= 1
    print()